.gitignore

```diff
@@ -106,4 +106,9 @@ build/
 .tern-port
 
 /app/poc_sample_payload.ts
-/app/sample_data.ts
\ No newline at end of file
+/app/sample_data.ts
+#all csv files
+*.csv
+
+#all DSstore files
+*.DS_Store
```

Comment:
Format and add new line after the end of file for all the files

Reply:
added